pkg/overlay/cache.go

```diff
@@ -79,6 +79,16 @@ func (cache *Cache) Inspect(ctx context.Context) (storage.Keys, error) {
 	return nil, errors.New("not implemented")
 }
 
+// DumpNodes returns all of the nodes in the overlay cache for inspector use
+func (cache *Cache) DumpNodes(ctx context.Context) ([]*pb.Node, error) {
+	n, err := cache.db.List(ctx, storj.NodeID{}, 0)
```

Comment:
This will list only the first 1,000 nodes: https://github.com/storj/storj/blob/master/satellite/satellitedb/overlaycache.go#L134-L136

Is this the intention?